libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/validator/VerifiableCredentialValidator.java

```diff
@@ -106,15 +104,10 @@ public VerifiableCredential parseAndValidate(
 
             var vc = VerifiableCredential.fromValidJwt(userId, criId, vcJwt);
 
-            if (configService.enabled(CoreFeatureFlag.PARSE_VC_CLASSES)) {
```

Comment:
I think this feature flag isn't used anymore - can it be dropped from CoreFeatureFlags?